source-browser.html

```diff
@@ -166,28 +191,60 @@ <h1 id="loading" class="m1 center display-none">⌛</h1>
               ),
             );
         });
-
+          
         $('#topics, #loading').toggleClass('display-none');
       }
-
+    
       function refreshTopics() {
         localStorage.clear();
         showTopics();
       }
-
+        
       function showSearchInput() {
         $('#search')
           .removeClass('display-none')
           .focus();
       }
-
+     
+        $(window).scroll(function() {
+            let top = $(window).scrollTop();
+            let bottom = $(window).scrollTop() + $(window).height();
+            for(let i = 0; i < LIST.length; i++){
+                let heading = LIST[i];
+                let vartop = $('#'+heading).position();
+                  if(top < vartop.top && vartop.top < bottom){
+                      $('#li'+heading).hide();
+              }
+                  else{
+                      $('#li'+heading).show();
+                      let $heading = $('#li'+heading);
+                      if(vartop.top>bottom){
```

Comment:
Spacing between binary operations here and throughout should be consistent with existing code.